integrationtests/tests/fixtures/mod.rs

```diff
@@ -105,6 +106,31 @@ pub struct Fixtures {
     pub task_group: TaskGroup,
 }
 
+pub async fn test<B>(
```

Comment:
Very generic name, and no comment. 🤪

Think about the children... Err.. I mean..., new contributors.

Reply:
Haha fair point, thought the code is self-explanatory.  I'll add a comment.

Reply:
The problem with code that is self-explanatory is that you still have to read the code to understand it first. A short docstring might save someone a lot of time to be able to tell "this is not what I'm looking for" without reading, when they are trying to understand / get something else to work.